README.md

```diff
@@ -1,5 +1,5 @@
 # operator-controller
-The operator-controller is the central component of Operator Lifecycle Manager (OLM) v1. It extends Kubernetes with an API through which users can install Operators.
+The operator-controller is the central component of Operator Lifecycle Manager (OLM) v1. It extends Kubernetes with an API through which users can install extensions.
```

Comment:
Tangentially related to this PR, should we consider renaming this repo to something more appropriate now that we are moving to the notion of life cycling extensions rather than operators?

Reply:
Yes we should discuss that, but that has _much_ wider ranging impacts, so I think that will require more planning/discussion than this PR does.

Reply:
100% agree that it needs much more planning/discussion. Just wanted to make sure it was called out that the name `operator-controller` is going to start being misleading (using this loosely) as we transition over to the "extension" nomenclature